README.md

```diff
@@ -57,6 +57,8 @@ it needs to be disabled.
 Download the archive matching your host OS from the Minishift [releases page](https://github.com/minishift/minishift/releases) and unpack it. Then copy the contained binary to your preferred
 location and optionally ensure it is added to your _PATH_.
 
+**Note**: For Windows OS, unpack the binary in your preferred location in the drive which contains your `%USERPROFILE%` directory. Mostly in `C:` drive.
+
```

Comment:
@praveenkumar , some issue which we inherited from minikube [issue #459](https://github.com/kubernetes/minikube/issues/459). It looks like we have to run binary from `%USERPROFILE%` only. Need to modify the note to add this too.

```
PS D:\> minishift.exe start --vm-driver virtualbox --show-libmachine-logs -v 5
Starting local OpenShift cluster...
Found binary path at C:\Users\budhram\home_bin\minishift.exe
Launching plugin server for driver virtualbox
Plugin server listening at address 127.0.0.1:50142
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
(minishift) Calling .GetMachineName
(minishift) Calling .DriverName
Running pre-create checks...
(minishift) Calling .PreCreateCheck
(minishift) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe --version
......
(minishift) Calling .GetConfigRaw
Creating machine...
(minishift) Calling .Create
(minishift) Downloading C:\Users\budhram\.minishift\cache\boot2docker.iso from file://C:/Users/budhram/.minishift/cache/iso/boot2docker.iso...
E0102 13:07:00.410154    1808 start.go:130] Error starting host: Error creating. Error creating machine: Error in driver during machine creation: open /Users/budhram/.minishift/cac
he/iso/boot2docker.iso: The system cannot find the path specified.. Retrying.
.....


PS D:\> which .\minishift.exe
/cygdrive/c/Users/budhram/home_bin/.\minishift.exe

PS D:\> minishift.exe version
minishift version: 1.0.0-beta.1

PS D:\> ls C:\Users\budhram\.minishift\cache\iso\
    Directory: C:\Users\budhram\.minishift\cache\iso
Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----       12/30/2016   8:02 PM       37748736 boot2docker.iso

```

### Switching to C Driver 
```
PS D:\> cd C:\Users\budhram\
PS C:\Users\budhram>

PS C:\Users\budhram> minishift.exe delete
Deleting local OpenShift cluster...
Machine deleted.

PS C:\Users\budhram> minishift.exe start --vm-driver virtualbox --show-libmachine-logs -v 5
Starting local OpenShift cluster...
Found binary path at C:\Users\budhram\home_bin\minishift.exe
Launching plugin server for driver virtualbox
Plugin server listening at address 127.0.0.1:50166
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
(minishift) Calling .GetMachineName
(minishift) Calling .DriverName
Running pre-create checks...
(minishift) Calling .PreCreateCheck
(minishift) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe --version
.........
(minishift) DBG | IPV6Address:
(minishift) Downloading C:\Users\budhram\.minishift\cache\boot2docker.iso from file://C:/Users/budhram/.minishift/cache/iso/boot2docker.iso...
........
(minishift) Creating VirtualBox VM...
(minishift) Creating SSH key...
........
```